lib/ecto/query/builder/from.ex

```diff
@@ -62,6 +62,9 @@ defmodule Ecto.Query.Builder.From do
           # When a binary is used, there is no model
           {1, query(source, nil)}
 
+        {source, model} when is_binary(source) and is_atom(model) ->
```

Comment:
@josevalim Hi, I have a question here.If I change this segment of code to this
```elixir
        {source, {:__aliases__, _, _} = module} when is_binary(source) ->
          {1, query(source, module)}
```
It seems work and the tests pass. Why?